dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/icon/IconRequestParams.java

```diff
@@ -53,6 +54,9 @@ public class IconRequestParams {
   private Date createdEndDate;
   private Date lastUpdatedStartDate;
   private Date lastUpdatedEndDate;
+  private boolean paging = true;
+  private int pageSize = Pager.DEFAULT_PAGE_SIZE;
+  private int page;
```

Comment:
```suggestion
  private int page = 1;
```

These fields are user-facing as they represent the query params. The defaults show up in OpenAPI specs. To users the first `page` starts at 1 no matter how we implement it internally.

Reply:
fixed